manifests/init.pp

```diff
@@ -699,6 +703,7 @@
   Optional[Variant[String,Array[String]]] $server_jvm_extra_args = $puppet::params::server_jvm_extra_args,
   Optional[String] $server_jvm_cli_args = $puppet::params::server_jvm_cli_args,
   Optional[Stdlib::Absolutepath] $server_jruby_gem_home = $puppet::params::server_jruby_gem_home,
+  Hash[String, String] $server_environment_vars = $puppet::params::server_environment_vars,
```

Comment:
Thinking out loud: Do we also want to accept `Integer` here? Users may write in hiera:
```yaml
puppet::server_environment_vars:
  myvar: 1
```

I think in Puppet this ends up as an Integer unless they write `"1"`.

Reply:
I was thinking from a technical perspective that environment variables are always strings by nature. But we can allow more types I you like and rely on implicit conversion during templating.